style.css

```diff
@@ -0,0 +1,67 @@
+@import url('https://fonts.googleapis.com/css2?family=Caveat:wght@700&family=Cuprum:wght@400;700&family=Kaushan+Script&display=swap');
+
+* {
+	box-sizing: border-box;
+}
+
+html {
+	background: url('imgs/2.jpg') no-repeat center center fixed;
+}
+
+body {
+	padding: 0;
+	margin: 0;
+	font-size: 62.5%;
+	font-family: "Cuprum", sans-serif;
+}
+
+.titleBar h1 {
+	text-align: center;
+	font-size: 6rem;
+	margin: 8px;
+	padding-top: 10px;
+}
+
+.titleBar p {
+	text-align: center;
+	font-family: 'Kaushan Script', cursive;
+	font-size: 3rem;
+	padding: 0;
+	margin: 0;
+	color: rgba(0,0,0,0.8);
+}
+
+.container {
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	font-size: 15px;
```

Comment:
Your font size in some places is in rems and in others in pixels. Maybe choose one unit of measure to use?